src/component/Select/Select.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Select = ({
```

Comment:
```suggestion
export const SelectUser = ({
```
потому что это специфический селектор, а не универсальный